src/tests/D3MHub.t.sol

```diff
@@ -1076,6 +1382,108 @@ contract D3MHubTest is DSSTest {
         assertEq(vat.dai(vow), vowDaiBefore + 50 * RAD);
     }
 
+    function test_cull_debt_paid_back_ignore_ilk_line() public {
+        _windSystem();
+
+        // Someone pays back our debt
+        _giveTokens(dai, 10 * WAD);
+        dai.approve(address(daiJoin), type(uint256).max);
+        daiJoin.join(address(this), 10 * WAD);
+        vat.frob(
+            ilk,
+            address(d3mTestPool),
+            address(d3mTestPool),
+            address(this),
+            0,
+            -int256(10 * WAD)
+        );
+
+        d3mHub.cage(ilk);
+
+        (uint256 pink, uint256 part) = vat.urns(ilk, address(d3mTestPool));
+        assertEq(pink, 50 * WAD);
+        assertEq(part, 40 * WAD);
+        assertEq(vat.gem(ilk, address(d3mTestPool)), 0);
+        uint256 sinBefore = vat.sin(vow);
+        uint256 vowDaiBefore = vat.dai(vow);
+
+        d3mHub.cull(ilk);
+
+        (uint256 ink, uint256 art) = vat.urns(ilk, address(d3mTestPool));
+        assertEq(ink, 0);
+        assertEq(art, 0);
+        assertEq(vat.gem(ilk, address(d3mTestPool)), 50 * WAD);
+        assertEq(vat.dai(address(d3mHub)), 0);
+        // Sin only increases by 40 WAD since 10 was covered previously
+        assertEq(vat.sin(vow), sinBefore + 40 * RAD);
+        assertEq(vat.dai(vow), vowDaiBefore);
+        (, , , uint256 culled, ) = d3mHub.ilks(ilk);
+        assertEq(culled, 1);
+
+        d3mTestPlan.file("bar", 10);
+        d3mTestPlan.file("targetAssets", 50 * WAD);
+        vat.file(ilk, "line", 40 * RAD);
```

Comment:
Mmm seems this and the test afterwards replicate `test_cull_debt_paid_back` but with 3 additionals lines each that actually make no difference since we are culled and will unwind as much as possible.
Suggest removing the 2 extra tests for the sake of readability.